tests/io_/read/test_tbl.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from peepingtom.io_.read.tbl import read_tbl
+
+external_table_files = Path() / 'test_data' / 'external' / 'tbl'
```

Comment:
This `Path()` assumes that working directory is something specific. We should use paths relative to `__file__`, I think.

Reply:
Okay I agree, and we should accept that upon refactoring tests will get broken. I vote we merge this now so it's there for the testing architecture PR